presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -554,7 +554,7 @@ public Void visitJoin(JoinNode node, Integer indent)
                         formatOutputs(node.getOutputSymbols()));
             }
 
-            node.getSortExpression().ifPresent(expression -> print(indent + 2, "SortExpression[%s]", expression));
+            node.getSortExpressionContext().ifPresent(context -> print(indent + 2, "SortExpression[%s]", context.getSortExpression()));
```

Comment:
Interesting what does it do in plan printer. Unconditionally to the flag enabling inequi join. (no change requested)